### PR TITLE
Today I learned: Elixir maps have built in syntax for updating

### PR DESCRIPTION
**Pull request by maabernethy**
Working title. Also, if we want to make this a real series of some sort, I can add more of an intro for "Today I learned" or whatever we want to call it.

### CODE REVIEW

**Comment by danmcclain** [comment]
I know we previously had reservations about using TIL since [HashRocket has a TIL series themselves](http://til.hashrocket.com)

**Comment by maabernethy** [comment]
@danmcclain Yeah I mentioned that in the eng meeting this morning. But Estelle said it shouldn't matter! But I am down to change it to "quick tips" or something else

**Comment by brzpegasus** [comment]
"Quick Tips" would work too!

File: source/posts/2016-03-07-til-elixir-update-synatx.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+layout: post
+title: "TIL: Elixir maps have built in syntax for updating"
+author: "Marin Abernethy"
+twitter: "pairinMarin"
+github: "maabernethy"
+published: true
+tags: elixir, til
+---
+
+I am relatively new to elixir and often find there are numerous ways to
```
Comment (brzpegasus) [inline source/posts/2016-03-07-til-elixir-update-synatx.md]: Capitalize `Elixir`

File: source/posts/2016-03-07-til-elixir-update-synatx.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+However, **today I learned**, maps come with a built in syntax for updating
+one or more key values!
+
+```ex
+ %{ expenses | groceries: 150, commute: 75}
+```
+
+While a bit obscure, and not discussed in the [elixir docs][docs], this trick is definitely nice to have in my elixir tool belt. The only thing to remember is that this syntax requires `groceries` and `commute`
```
Comment (bcardarella) [inline source/posts/2016-03-07-til-elixir-update-synatx.md]: I would confirm this, might want to ask in #elixir-lang on IRC if this is true. I've definitely seen this used in the docs for Ecto but I'm not sure if it is covered in the guides/docs anywhere

**Comment by maabernethy** [comment]
Reminder for myself: update url after deciding title! :fish:

File: source/posts/2016-03-07-til-elixir-update-synatx.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+```elixir
+expenses = %{groceries: %{milk: 5}, apartment: %{rent: 1000, comcast: 100}}
+put_in(expenses, [:rent, :comcast], "too much")
+```
+
+However, **today I learned**, maps come with a built in syntax for updating
+one or more key values!
+
+```ex
+ %{ expenses | groceries: 150, commute: 75}
```
Comment (poteto) [inline source/posts/2016-03-07-til-elixir-update-synatx.md]: There's an extra leading whitespace here!

**Comment by poteto** [comment]
Also NBD but `s/til-elixir-update-synatx.md/til-elixir-update-syntax.md`

**Pull request merged**